src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -92,6 +92,36 @@ override void OnLaunched(LaunchActivatedEventArgs e)
 				// this.DebugSettings.EnableFrameRateCounter = true;
 			}
 #endif
+			InitializeFrame(e);			
+			Windows.UI.Xaml.Window.Current.Activate();
+
+			DisplayLaunchArguments(e);
+		}
+
+		protected
+#if HAS_UNO
+			internal
+#endif
+			override async void OnActivated(IActivatedEventArgs e)
+		{
+			base.OnActivated(e);
+
+			InitializeFrame(e);		
+			Windows.UI.Xaml.Window.Current.Activate();
+
+			if (e.Kind == ActivationKind.Protocol)
+			{
+				var protocolActivatedEventArgs = (ProtocolActivatedEventArgs)e;
+				var dlg = new MessageDialog(
+					$"PreviousState - {e.PreviousExecutionState}, " +
+					$"Uri - {protocolActivatedEventArgs.Uri}",
+					"Application activated via protocol");
+				await dlg.ShowAsync();
+			}
+		}
+
+		private Frame InitializeFrame(IActivatedEventArgs e)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change return type to 'void'; not a single caller uses the returned value.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5378714)